rocky/rocky/templates/partials/form/field_input_radio.html

```diff
@@ -1,24 +1,25 @@
-<div>
-    {{ field.label_tag }}
+<fieldset>
+  <legend>
+    {{ field.label }}
+  </legend>
+  {% for choice in field %}
+      <div class="horizontal-view">
```

Comment:
This is within a form with the horizontal-view class right? If tat's the case, we shouldn't need this horizontal-view for each object. The styling should work based on the fact that the content is grouped within a div.

If it's added for checkbox or radio-button alignment I would suggest adding the "checkbox" and "radio" classes instead. For examples see: 
-https://minvws.github.io/nl-rdo-manon/documentation/components/checkbox.html  
-https://minvws.github.io/nl-rdo-manon/documentation/components/fieldset-radio.html